client/src/pages/tasks.html

```diff
@@ -0,0 +1,34 @@
+<template>
+  <require from="./tasks.css"></require>
+  <require from="../elements/schema-editor/schema-editor"></require>
+  <q-bar if.bind="user.isLoggedIn" router.bind="router">
+    <a slot="back-button" class="q-bar__back" route-href="route: index">
+      <button-secondary icon="back" icon-size="medium"></button-secondary>
+    </a>
+    <icon-logo class="index__logo" if.bind="enoughNewItems" slot="logo" click.delegate="reloadItems()"></icon-logo>
+    <icon-m class="index__logo" if.bind="!enoughNewItems" slot="logo" click.delegate="reloadItems()"></icon-m>
+    <language-switcher slot="language-switcher"></language-switcher>
+    <button-secondary slot="account-button" if.bind="user.isLoggedIn" click.delegate="showAccountModal()">${user.data.initials}</button-secondary>
+  </q-bar>
+  <section class="tasks">
+    <div id="tasks--sidebar">
```

Comment:
please use a class here, not an id. And change the name to `tasks__sidebar` according to BEM. This is a child-block, not a modifier. `--` is only used for modifier classes.

The same applies to the other child elements of `.tasks`.